packages/salesforcedx-vscode-core/src/commands/forceApexExecute.ts

```diff
@@ -38,29 +33,6 @@ class ForceApexExecuteExecutor extends SfdxCommandletExecutor<{}> {
       .withLogName('force_apex_execute')
       .build();
   }
-
-  public execute(response: ContinueResponse<TempFile>): void {
```

Comment:
@muenzpraeger - Was this just dead code then?

Reply:
Gotcha. I think that's actually useful to persist for debugging and, perhaps, an "Execute last command" that can persist throughout VS Code sessions.

But, let me not get ahead of myself :-) There is actually an official way for extensions to store information that could persist across sessions. See https://code.visualstudio.com/docs/extensionAPI/vscode-api#ExtensionContext if you are interested.